### ISSUE TITLE
resolving rule conflicts

### ISSUE THREAD

**Issue by Hifilo** [elirasza/stylelint-stylistic#9]
Hello,

I have a rule conflict that seems to occur on save. I have stylelint configured for css files, and I'm also using eslint & prettier.

![Code_G6vUGrZUOX](https://user-images.githubusercontent.com/92943726/233805428-ddbb5bbb-20cc-4fed-a7f8-3567805768b2.gif)

Is there a way for me to determine the source/cause of the conflict?

my .stylelintrc.json:

```json
{
    "plugins": ["stylelint-stylistic"],
    "extends": [
        "stylelint-config-standard",
        "stylelint-stylistic/config"
    ],
    "rules": {
        "property-no-vendor-prefix": [
            true,
            {
                "severity": "warning"
            }
        ],
        "font-family-name-quotes": null,
        "color-named": "never",
        "stylistic/declaration-block-semicolon-newline-after": "always",
        "stylistic/function-parentheses-space-inside": "always",
        "stylistic/value-list-comma-newline-after": "always",
        "stylistic/max-empty-lines": [2, { "ignore": ["comments"] }],
        "custom-property-empty-line-before": ["always", { "except": ["after-comment"] }],
        "stylistic/string-quotes": "single"
    }
}
```

**Comment by elirasza** [comment]
Hello ! I think it has to do with prettier, who has its own indentation rules. Try disabling the `stylelint/indentation` rule, and manually set a wrong indentation to see who's complaining ; what will happen ?

**Comment by Hifilo** [comment]
> Hello ! I think it has to do with prettier, who has its own indentation rules. Try disabling the `stylelint/indentation` rule, and manually set a wrong indentation to see who's complaining ; what will happen ?

Thanks @elirasza,
setting `"indentation": null` ,`"stylelint/indentation": null` , had no effect.
so I set `"stylistic/indentation": null` which removed the error. Indentation on save would autofix itself, but when I change the indentation manually I see no feedback in the problems panel.

If it makes any difference I have applied `"stylelint-config-prettier"`, but I don't see any difference with or without it.


![Code_sxuC49hPma](https://user-images.githubusercontent.com/92943726/234103077-a00128a5-0cc4-4cc5-8f9c-784a83a22375.png)

**Comment by elirasza** [comment]
Oh, it seems the `stylelint/indentation` rule was already deprecated in the main package then. Using `stylistic/indentation` to disable the rule for your needs seems the best way to go then. As for `stylelint-config-prettier`, I think its goal was to remove conflicts with the old stylelint (before 15.0.0) and prettier ; it seems to be discontinued for now. Maybe the config author should update its rules to target `stylelint-stylistic` ?

**Comment by Hifilo** [comment]
> Oh, it seems the `stylelint/indentation` rule was already deprecated in the main package then. Using `stylistic/indentation` to disable the rule for your needs seems the best way to go then. As for `stylelint-config-prettier`, I think its goal was to remove conflicts with the old stylelint (before 15.0.0) and prettier ; it seems to be discontinued for now. Maybe the config author should update its rules to target `stylelint-stylistic` ?

Okay I see. I've found myself a little lost in the workflows/setups/configs etc, so yeah I assumed it was something related to my setup. With that said, I have my linters running well I don't want to do anything to break it 😭.

I will confirm the next time I'm working in a css file, and set up stylelint.

Thank you.

**Comment by Hifilo** [comment]
Edit: I resolved it with `"stylistic/function-parentheses-space-inside": null`

Cheers

Hello. Reopening, because I've started using stylelint again. I'm not using `stylelint-config-prettier` I've got most errors sorted with the exception of this one:

`Unexpected whitespace before ")" (stylistic/function-parentheses-space-inside)`


`"stylistic/function-parentheses-space-inside": "always"`
![Code_aVgKATAqQz](https://github.com/elirasza/stylelint-stylistic/assets/92943726/8d4d60ed-267a-48f9-8675-20f6605098d2)

`"stylistic/function-parentheses-space-inside": "never"`

![Code_zFs6Rk1VHN](https://github.com/elirasza/stylelint-stylistic/assets/92943726/47550d46-d836-4b14-867f-036090b74048)

There doesn't appear to be any whitespace but its still getting flagged.


![Code_whkbs2X7lu](https://github.com/elirasza/stylelint-stylistic/assets/92943726/f68a8aef-3116-4453-aa22-c6ab6f0a4b7c)

any idea where the conflict might be?
